website/content/docs/security/acl/auth-methods/index.mdx

```diff
@@ -136,3 +136,30 @@ bearer token for a Consul ACL token by using the login process:
 
 For more details about specific auth methods and how to configure them, click
 on the name of the auth method type in the sidebar.
+
+# Token Locality
+
+By default, Auth Methods create ACL tokens with _local_ scope. ACL tokens with
+local scope are only valid in the local datacenter where they were created.
+Auth Methods created in the primary datacenter can be configured to create ACL
+tokens that have global scope; however, there are limitations depending on the
+type of the Auth Method being used. For all auth methods, setting
+[`-token-locality=global`](/commands/acl/auth-method/create#token-locality) for
+requests to secondary datacenters will fail. 
+
+For cross-datacenter requests that use a token provided by an Auth Method, one
+of the following must be true for requests to be successfully authenticated:
+- the Auth Method is configured with [`-token-locality=global`](/commands/acl/auth-method/create#token-locality)
+- the anonymous token policy in the remote datacenter grants access to read services
+
+Depending on the type of Auth Method being used, configuring it with
+[`-token-locality=global`](/commands/acl/auth-method/create#token-locality) may
+not be possible. Alternatively, the operator can configure the Auth Method and
+associated Binding Rules manually in all datacenters as a workaround.
```

Comment:
```suggestion
associated binding rules manually in all datacenters as a workaround.
```
As stated above - unless there's a compelling reason, I don't think we should treat auth(entication) methods as proper nouns, either.